cmd/nfd-worker/main.go

```diff
@@ -104,7 +104,8 @@ func initFlags(flagset *flag.FlagSet) (*worker.Args, *worker.ConfigOverrideArgs)
 	flagset.StringVar(&args.KeyFile, "key-file", "",
 		"Private key matching -cert-file")
 	flagset.BoolVar(&args.EnableNodeFeatureApi, "enable-nodefeature-api", true,
-		"Enable the NodeFeature CRD API for communicating with nfd-master. This will automatically disable the gRPC communication.")
+		"Enable the NodeFeature CRD API for communicating with nfd-master. This will automatically disable the gRPC communication."+
+			"DEPRECATED: will be removed in a future release.")
```

Comment:
...same here:

```suggestion
			"DEPRECATED: will be removed in a future release along with the deprecated gRPC API")
```

WDYT?